velox/expression/fuzzer/ExpressionFuzzer.cpp

```diff
@@ -451,6 +451,7 @@ bool ExpressionFuzzer::isSupportedSignature(
   if (usesTypeName(signature, "opaque") ||
       usesTypeName(signature, "interval day to second") ||
       usesTypeName(signature, "ipprefix") ||
+      usesTypeName(signature, "sfmsketch") ||
```

Comment:
nit: Could you add the github issue link as a comment for this skip so that we don't forget it?